client/src/components/to-do-list/Categories.jsx

```diff
@@ -3,9 +3,15 @@
 import React from 'react';
 import Category from './Category.jsx';
 
-function Categories({taskData, addTask, deleteTask}) {
+function Categories({addTask, openModal, categorizedTasks, editClick, editing, deleteTask}) {
```

Comment:
This is going to get ridiculous pretty soon with a dozen params being passed in. Consider an options object and destructure from there